File: Python/armstrong.py
n = input("Enter the number: ")
l=len(n)
s=0
temp=int(n)
while(temp>0):
    d = temp%10
    d=d**l
    s+=d
    temp=temp//10
if(int(n)==s):
    print(n," is a Armstrong Number")
else:
    print(n,"is Not a Armstrong Number")

File: armstrong.cpp
#include<iostream>
#include<math.h>
using namespace std;
int main()
{
    int n,l,O;
    int sum=0;
    cout<<"ENTER NUMBER: ";
    cin>>n;
    O=n;
    while(n>0)
    {
        l=n%10;
        sum=sum+pow(l,3);
        n=n/10;
    }
      if(sum=O)
      {
          cout<<"Armstrong";
      }
      else
      {
          cout<<"Not Armstrong";
      }


    return 0;
}
